src/variables.jl

```diff
@@ -1543,10 +1543,22 @@ function _info_from_variable(v::VariableRef)
 end
 
 """
-    relax_integrality(model::Model)
+    relax_integrality(model::Model; fix::Union{Nothing,Function} = nothing)
 
 Modifies `model` to "relax" all binary and integrality constraints on
-variables. Specifically,
+variables. In addition, if `fix` is passed a function, then the discrete
+variables are fixed to the value of `fix(x)`.
```

Comment:
If `fix` returns `nothing` then the variable is not fixed or it throws an error ?

Reply:
Rewritten as two separate functions.